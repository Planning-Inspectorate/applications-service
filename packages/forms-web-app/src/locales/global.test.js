const enGlobalTranslations = require('./en/global.json');
const cyGlobalTranslations = require('./cy/global.json');

describe('locales/global', () => {
	it('should return the correct English global translations', () => {
		expect(enGlobalTranslations).toMatchSnapshot();
	});

	it('should return the correct Welsh global translations', () => {
		expect(cyGlobalTranslations).toMatchSnapshot();
	});

	it('should return the same keys for English and Welsh translations', () => {
		expect(enGlobalTranslations).toHaveSameKeys(cyGlobalTranslations);
	});
});
