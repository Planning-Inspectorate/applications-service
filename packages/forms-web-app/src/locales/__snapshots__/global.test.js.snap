// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`locales/global should return the correct English global translations 1`] = `
{
  "backLinkText": "Back",
  "betaBanner": {
    "content": "This is a beta service - your {{-link}} will help us to improve it",
    "feedback": "feedback",
  },
  "cookieBanner": {
    "acceptButtonText": "Accept analytics cookies",
    "acceptedCookiesText": "You’ve accepted analytics cookies. You can {{-link}} at any time.",
    "actionCookiesLinkText": "change your cookie settings",
    "cookieBannerAriaLabel": "cookie banner",
    "headingDefaultService": "application service",
    "headingPrefix": "Cookies on",
    "hideMessage": "Hide this message",
    "paragraph1": "We use some essential cookies to make this service work.",
    "paragraph2": "We'd also like to use analytics cookies so we can understand how you use the service and make improvements.",
    "rejectButtonText": "Reject analytics cookies",
    "rejectedCookiesText": "You’ve rejected analytics cookies. You can {{-link}} at any time.",
    "viewCookiesLinkText": "View cookies",
  },
  "footer": {
    "content": "All content is available under the {{-link}}, except where otherwise stated",
    "contentLinkText": "Open Government Licence v3.0",
    "copyright": "Crown copyright",
    "links": {
      "accessibility": "Accessibility statement",
      "contact": "Contact",
      "cookies": "Cookies",
      "privacy": "Privacy",
      "t&c": "Terms and conditions",
    },
  },
  "headerTitle": {
    "default": "Find a National Infrastructure Project",
    "examination": "Have your say on an application",
    "getUpdates": "Get updates about this project",
    "register": "Register to have your say",
  },
  "paginationNavigation": "Pagination navigation",
  "primaryNavigation": {
    "detailedInformation": "Detailed information",
    "home": "Home",
    "primaryNavigationAria": "Primary navigation",
    "projectSearch": "All projects",
  },
  "relatedContentNavigation": "Related content navigation",
}
`;

exports[`locales/global should return the correct Welsh global translations 1`] = `
{
  "backLinkText": "Yn ôl",
  "betaBanner": {
    "content": "Gwasanaeth beta yw hwn - bydd eich {{-link}} yn ein helpu i'w wella.",
    "feedback": "adborth",
  },
  "cookieBanner": {
    "acceptButtonText": "Accept analytics cookies ** WELSH **",
    "acceptedCookiesText": "You’ve accepted analytics cookies. You can {{-link}} at any time.  ** WELSH **",
    "actionCookiesLinkText": "change your cookie settings ** WELSH **",
    "cookieBannerAriaLabel": "cwci baner",
    "headingDefaultService": "gwasanaeth ymgeisio",
    "headingPrefix": "Cwcis ar y",
    "hideMessage": "Hide this message ** WELSH **",
    "paragraph1": "We use some essential cookies to make this service work. ** WELSH **",
    "paragraph2": "We'd also like to use analytics cookies so we can understand how you use the service and make improvements. ** WELSH **",
    "rejectButtonText": "Reject analytics cookies ** WELSH **",
    "rejectedCookiesText": "You’ve rejected analytics cookies. You can {{-link}} at any time.  ** WELSH **",
    "viewCookiesLinkText": "View cookies ** WELSH **",
  },
  "footer": {
    "content": "Mae'r holl gynnwys ar gael o dan {{-link}}, ac eithrio lle nodir yn wahanol",
    "contentLinkText": "Drwydded Llywodraeth Agored v3.0",
    "copyright": "Hawlfraint y Goron",
    "links": {
      "accessibility": "Datganiad hygyrchedd",
      "contact": "Cysylltu â ni",
      "cookies": "Cwcis",
      "privacy": "Preifatrwydd",
      "t&c": "Telerau ac amodau",
    },
  },
  "headerTitle": {
    "default": "Dod o hyd i Brosiect Seilwaith Cenedlaethol",
    "examination": "Lleisio'ch barn am gais",
    "getUpdates": "Cael diweddariadau ynglŷn â'r prosiect hwn",
    "register": "Cofrestru i leisio'ch barn",
  },
  "paginationNavigation": "tudalen cyfeiri",
  "primaryNavigation": {
    "detailedInformation": "Gwybodaeth fanwl",
    "home": "Hafan",
    "primaryNavigationAria": "Cyfeiri cynradd",
    "projectSearch": "Pob prosiect",
  },
  "relatedContentNavigation": "Cyfeiri cynnwys cysylltiedig",
}
`;
